ghcide/src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -205,7 +205,7 @@ atPoint
 atPoint IdeOptions{} (HAR _ hf _ _ kind) (DKMap dm km) env pos = listToMaybe $ pointCommand hf pos hoverInfo
   where
     -- Hover info for values/data
-    hoverInfo ast = (Just range, prettyNames ++ pTypes)
+    hoverInfo ast = (Just range, prettyNames ++ [] ++ pTypes)
       where
```

Comment:
Here is a Lint warning:
![Screenshot-2021-12-25-01:54:48](https://user-images.githubusercontent.com/20933385/147374440-e63fb3ed-1b39-4853-8581-8aaf36e8a08e.png)

Reply:
Kept this open to show workflow works.